app/models/user-message.ts

```diff
@@ -0,0 +1,10 @@
+// app/models/user-message.js
+import Model, { attr, belongsTo } from '@ember-data/model';
+
+export default class UserMessageModel extends Model {
+  @attr('string') messageText;
+  @attr('string') messageType;
```

Comment:
Are there pre-defined `messageType` values? This could be typed as an enum if so, by adding something like this to the top of this file:
```
export enum MessageTypeChoices {
    ChoiceOne = 'choice_one',
    ChoiceTwo = 'choice_two',
}
```

then doing
```
@attr('string') messageType: MessageTypeChoices;
```